android/app/src/main/java/com/microsoft/codepush/react/CodePushTelemetryManager.java

```diff
@@ -3,18 +3,28 @@
 import android.content.Context;
 import android.content.SharedPreferences;
 
+import com.facebook.react.bridge.ReadableMap;
 import com.facebook.react.bridge.WritableMap;
 import com.facebook.react.bridge.WritableNativeMap;
 
+import org.json.JSONException;
+import org.json.JSONObject;
+
 public class CodePushTelemetryManager {
 
     private Context applicationContext;
+    private final String APP_VERSION_KEY = "appVersion";
```

Comment:
usually you make these things static as well, since they're shared across every CodePushTelemetryManager.

that may require you to add "CodePushTelemetryManager." before every use but I don't think so